molecule/accessible-namespaces-test/converge.yml

```diff
@@ -6,9 +6,9 @@
   vars:
     custom_resource: "{{ lookup('template', cr_file_path) | from_yaml }}"
   tasks:
-  - import_tasks: ../common/tasks.yml
-  - import_tasks: ../asserts/pod_asserts.yml
-  - import_tasks: ../asserts/accessible_namespaces_contains.yml
+  - include_tasks: ../common/tasks.yml
+  - include_tasks: ../asserts/pod_asserts.yml
+  - include_tasks: ../asserts/accessible_namespaces_contains.yml
```

Comment:
Assuming the numerous changes from import (static) to include (dynamic) are all intentional.

Reply:
yes. intentional